.gitignore

```diff
@@ -6,3 +6,5 @@
 /dist/
 /build/
 docs/_build
+.idea/
+.python-version
```

Comment:
please don't edit .gitignore files on projects you don't maintain -- you can put these in a personal global gitignore (please rebase this out)

Reply:
Alright! It was only a suggestion hence the distinct commit 😉